aws/diff_suppress_funcs.go

```diff
@@ -102,3 +102,14 @@ func suppressAutoscalingGroupAvailabilityZoneDiffs(k, old, new string, d *schema
 func suppressRoute53ZoneNameWithTrailingDot(k, old, new string, d *schema.ResourceData) bool {
 	return strings.TrimSuffix(old, ".") == strings.TrimSuffix(new, ".")
 }
+
```

Comment:
Inspired by https://github.com/terraform-providers/terraform-provider-aws/pull/5508#discussion_r210787219.
I'll create an issue to refactor other uses.